debian/copyright

```diff
@@ -0,0 +1,22 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: neofs-node
+Upstream-Contact: tech@nspcc.ru
+Source: https://github.com/nspcc-dev/neofs-node
+
+Files: *
+Copyright: 2018-2022 NeoSPCC (@nspcc-dev), contributors of neofs-go project (https://github.com/nspcc-dev/neofs-s3-gw/blob/master/CREDITS.md)
```

Comment:
what does it mean neofs-go project? And why is here link to the s3 gateway repository?
Looks like need to change this "contributors of neofs-go project (https://github.com/nspcc-dev/neofs-s3-gw/blob/master/CREDITS.md) to "contributors of neofs-node project ( https://github.com/nspcc-dev/neofs-node/blob/master/CREDITS.md )"

Reply:
sorry typos :-(